releases/release-5.0.1.md

```diff
@@ -0,0 +1,86 @@
+---
+title: TiDB 5.0.1 Release Notes
+---
+
+# TiDB 5.0.1 Release Notes
+
+Release date: April 23, 2021
+
+TiDB version: 5.0.1
+
+## Compatibility Changes
+
+## New Features
+
+## Improvements
+
```

Comment:
+Tools
   - BR
       * * Fix the issue that the summary log is not clear. [#1009](https://github.com/pingcap/br/pull/1009)